app/javascript/app/pages/national-context/sectoral-activity/selectors/sectoral-activity-constants.js

```diff
@@ -15,11 +16,17 @@ export const SECTION_COLORS = {
   ENERGY: '#2EC9DF',
   FORESTRY: '#FCA683',
   INDUSTRIAL_PROCESS_AND_PRODUCT_USE: '#FFC735',
-  'no-data': '#ffffff'
+  'no-data': '#FAFAFA'
```

Comment:
My :eye: don't see a difference between this and first bucket white color. But as a :mens: I could be just color blind.

Reply:
You're right, that is a bit unclear, for a 🚺 as well 😂 I'm gonna check with Fausto what does he think is the best color for no-data.